lib/io/i2c.cpp

```diff
@@ -4,6 +4,7 @@
 #include <unistd.h>
 
 #include <sys/ioctl.h>
+#include <stdio.h>
```

Comment:
`cstdio` is already included in the header for this file and also contains a definition of `snprintf` so this include is not necessary.